sadcowpng/sharedspaces/static/account.css

```diff
@@ -22,10 +22,20 @@
         height: 800px;
     }
 
+    .badge {
+        background-color: #F4976C;
+    }
+
+    span {
+        color: grey;
+        font-size: 20px;
+    }
+
```

Comment:
Might mess up the closed and open badges. We might want to keep an eye out for that before we merge into main later on.

Reply:
Yeah, I was like "oh no". We will see once we pull from develop.